app/views/providers/confirm_dwp_non_passported_applications/show.html.erb

```diff
@@ -1,25 +1,33 @@
-<% error_class = @error ? 'alternative-error-style' : '' %>
+<% error_class = @form.errors.any? ? 'alternative-error-style' : '' %>
 
 <div class="interruption-panel">
-  <%= page_template(page_title: t('.tab_title'), column_width: :full, template: :basic) do %>
+  <%= page_template(page_title: t('.tab_title'), column_width: :full, show_errors_for: nil, template: :basic) do %>
+
+    <!-- duplicate the model but remove errors so original error styling is removed-->
+    <%
+      model = @form.dup
+      model.errors.delete(:confirm_dwp_non_passported_application)
+    %>
 
     <%= form_with(
           builder: GOVUKDesignSystemFormBuilder::FormBuilder,
-          model: @form,
+          model: model,
           url: providers_legal_aid_application_confirm_dwp_non_passported_applications_path,
           method: :patch,
           local: true
         ) do |form| %>
 
       <div class="<%= "#{error_class}" %>">
-        <%= form.govuk_radio_buttons_fieldset(:dwp_results_correct, legend: {text: t('.title'), tag: 'h1', size: 'l'} ) do %>
-          <%= form.govuk_radio_button(:dwp_results_correct, true, label: {text: t('generic.yes')} ) %>
-          <%= form.govuk_radio_button(:dwp_results_correct, false, label: {text:  t('.no')} ) %>
-        <% end %>
+        <%= form.govuk_collection_radio_buttons :correct_dwp_result,
+                                                [OpenStruct.new(value: true, label: t('generic.yes')),
+                                                 OpenStruct.new(value: false, label: t('.no'))],
+                                                :value,
+                                                :label,
+                                                legend: {text: content_for(:page_title), tag: 'h1', size: 'l'} %>
       </div>
 
-      <% if @error %>
-        <%= link_to_accessible @error.values.first, "#dwp-results-correct-true-field", class: 'govuk-heading-s' %>
+      <% if @form.errors.any? %>
+        <%= link_to_accessible @form.errors.values.first, "#confirm_dwp_non_passported_application-true-field", class: 'govuk-heading-s' %>
```

Comment:
By changing 'dwp-results-correct-true-field' to 'confirm_dwp_non_passported_application-true-field' the link no longer takes the user to the relevant field. The GovUk design says that clicking the error message should move the focus to the relevant field, so in this case we should move focus to the Yes radio button.

Reply:
Yeah not sure how this got here, pushing fix now